pom.xml

```diff
@@ -1591,7 +1592,12 @@
             <dependency>
                 <groupId>com.yahoo.platform.yui</groupId>
                 <artifactId>yuicompressor</artifactId>
-                <version>2.4.7</version>
+                <version>2.4.8</version>
+            </dependency>
+            <dependency>
+                <groupId>com.google.javascript</groupId>
+                <artifactId>closure-compiler</artifactId>
+                <version>${closure.compiler.version}</version>
```

Comment:
I think this dependency and/or the YUI compressor should be optional dependency. I'm leaning towards just the closure-compiler being optional. This will give us a way to use @Conditionals to determine if we're using the closure-compiler to minify JS.